README.md

```diff
@@ -84,8 +84,8 @@ $ 127.0.0.1:6379> HEXISTS OP_KEYS_ALLOWLIST "0734FAC43DBE455D531930B6A8E02404335
 $ (integer) 1 
 ```
 
-**Prepare ownership voucher keys denylist**:
-* Create Redis Hash **OP_KEYS_DENYLIST**
+**Prepare Ownership Voucher Keys Denylist**:
+* Create Redis Hash **OP_KEYS_DENLYLIST**
```

Comment:
DENLYLIST?  I think this is a typo.

Reply:
@grbean The term BLACKLIST has been replaced with DENYLIST in the source code.

Reply:
Fixed.